boards/arm/nrf7002dk_nrf5340/nrf5340_cpuapp_common.dts

```diff
@@ -115,32 +116,32 @@
 arduino_i2c: &i2c1 {
 	compatible = "nordic,nrf-twim";
 	status = "okay";
-	sda-pin = <34>;
-	scl-pin = <35>;
+	pinctrl-0 = <&i2c1_default>;
+	pinctrl-1 = <&i2c1_sleep>;
+	pinctrl-names = "default", "sleep";
```

Comment:
This is a fixup commit. We do not accept fixup commits in NCS, because we follow Zephyr's contribution guides. Please rebase the fix as needed.

If you are unfamiliar with this terminology, this 30 minute presentation explains the basic expectations around branch structure, including fixup commits: https://www.youtube.com/watch?v=mnVNLM7ynMQ

Reply:
Well, this was unintentional by mistake during rebase wrong parent commit for fixup was done. Will fix this.